python/GafferUI/CompoundEditor.py

```diff
@@ -1682,7 +1680,7 @@ def __getBookmarkSet( self ) :
 		return None
 
 	def __leave( self, widget ) :
-
+		# TODO - does this function do anything?
```

Comment:
It doesn't look like it. I think it should have been removed completely in 0658b4c1042b0fa9db28739539142d3e1aefb07b, which removed the one bit of it that used to do anything. Let's get rid.